metron-interface/metron-config/scripts/package.json

```diff
@@ -18,5 +18,5 @@
   },
   "private": true,
   "author": "",
-  "license": "MIT"
+  "license": "Apache 2.0"
```

Comment:
Looking at https://docs.npmjs.com/files/package.json and https://spdx.org/licenses/, it looks like this should probably be `Apache-2.0` (with the dash).  Is my understanding of this correct?